bokeh/io.py

```diff
@@ -559,3 +563,92 @@ def _destroy_server(div_id):
 
     except Exception as e:
         logger.debug("Could not destroy server for id %r: %s" % (div_id, e))
+
+def export(obj, filename=None):
+    ''' Save an HTML file with the data for the current document.
+
+    If the filename is not given, it is derived from the script name
+    (e.g. ``/foo/myplot.py`` will create ``/foo/myplot.png``)
+
+    Args:
+        obj (LayoutDOM object) : a Layout (Row/Column), Plot or Widget object to display
+
+        filename (str, optional) : filename to save document under (default: None)
+            If None, infer from the filename.
+
+    Returns:
+        filename (str) : the filename where the static file is saved.
+
+    '''
+
+    # probably need a specific version
+    webdriver = import_required('selenium.webdriver',
+                                'To use bokeh.io.export you need selenium ' +
+                                '("conda install -c bokeh selenium" or "pip install selenium")')
+
+    # assert that phantomjs is in path for webdriver
+    detect_phantomjs()
+
+    driver = webdriver.PhantomJS()
+
+    # generate the html as tempfile
+    temp_html = tempfile.NamedTemporaryFile(suffix=".html").name
+    save(obj, filename=temp_html, resources=INLINE, title="")
+
+    if filename is None:
+        filename = _detect_filename("png")
+
+    driver.get("file:///" + temp_html)
```

Comment:
I am already forcing inline resources for the html tempfile that's being generated. I'll add this to my list of general questions though (is forcing inline resources bad?)